webroot/src/models/Licensee/Licensee.model.ts

```diff
@@ -230,8 +230,14 @@ export class LicenseeSerializer {
         };
 
         // In get-all responses, server only returns a license state, not the actual license objects
```

Comment:
Is this comment still true?

Reply:
Confirming with the team but I think there actually weren't changes here. However, Im glad you pointed this out because I should not have touched this file and must have forgotten to clean up after I was prototyping certain things. As a result I have moved a portion of the functionality to the UI presentation layer where it should have been from the beginning.

Reply:
Yea both cases still need to be there